app/services/inherited_proofing/va/service.rb

```diff
@@ -0,0 +1,66 @@
+require_relative 'service_payload_validator'
+
+module InheritedProofing
+  module Va
+    # Encapsulates request, response, error handling, validation, etc. for calling
+    # the VA service to gain PII for a particular user that will be subsequently
+    # used to proof the user using inherited proofing.
+    class Service
+      REQUEST_URI = 'https://staging-api.va.gov/inherited_proofing/user_attributes'.freeze
+
+      attr_reader :auth_code
+
+      def initialize(auth_code)
+        @auth_code = auth_code
+      end
+
+      # Calls the endpoint and returns the decrypted response.
+      def execute
+        response = Net::HTTP.get_response(request_uri, request_headers)
```

Comment:
For better or worse we use [Faraday](https://github.com/lostisland/faraday) to make external HTTP requests in the IdP